### ISSUE TITLE
Matchit breaks `[operator]V%`

### ISSUE THREAD

**Issue by bstaletic** [vim/vim#3720]
When `matchit.vim` is loaded things like `dV%` or `cV%` stop working.

Steps to reproduce:
- `vim -u NONE`
- `:packadd matchit`
- Put the following into the buffer:
```
if () {

} // junk
```
- Place the cursor on `{`
- `dV%`

Expected behaviour - the whole buffer is cleared.
Actual behaviour - Same as `d%` - `if ()  // junk`.

Vim version: 8.1.570
Matchit header:
```viml
"  matchit.vim: (global plugin) Extended "%" matching
"  Last Change: 2018 Jul 3 by Christian Brabandt
"  Maintainer:  Benji Fisher PhD   <benji@member.AMS.org>
"  Version:     1.13.3, for Vim 6.3+
"		Fix from Fernando Torres included.
"		Improvement from Ken Takata included.
```

**Comment by k-takata** [comment]
Maybe same as chrisbra/matchit#12?
#3490 might be also related.

**Comment by bstaletic** [comment]
Yes, chrisbra/matchit#12 looks like the same thing. Should this issue be closed?

**Comment by chrisbra** [comment]
closing.

**Comment by chrisbra** [comment]
To have this fixed, we need vim/vim#3490 included.
